acc-external-testcases/resources/load_balancer_profile-profile_err.yaml

```diff
@@ -0,0 +1,28 @@
+vars:
+  http_name: tf_HTTP_PROFILE_%rand_int
+acc:
+- config: |
+    lb_id = 18 
+    name  =  "$(http_name)"      
+    description  = "HTTP_PROFILE creating using tf"
+    profile_type = "application-profile"
+    http_profile {
```

Comment:
why these test cases coming in ? We discussed that we need to handle more than one NSX integration usecase to make IaC work for external CLoud with NSX as networking platform

Reply:
i had simply duplicated it, will remove it